infra/gp-external-secrets-configuration/templates/cicd-development-store.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ClusterSecretStore
+metadata:
+  name: cicd-development-store
```

Comment:
cicd-secret-store?

Reply:
Werd mir sprechendere Namen für die beiden Stores überlegen

Reply:
sie heißen jetzt 'cicd-internal-cluster-store' für den nur gepaplexx-intern verwendbaren Store und 'cicd-project-specific-cluster-store' für den, der nur Secrets aus dem Pfad mit selbem Namen wie der Namespace, aus dem der Call kommt zurückgibt

Reply:
vielleicht nur "project" store? aber sonst passt es von meiner seite

Reply:
mmh, find ich nicht so gut. Ich will in dem Namen vor allem ausdrücken, was da im Vault berechtigungstechnisch dahinter steht, weil das in der CR sonst überhaupt nicht erkennbar ist. Wenn ich wem erklär welcher verwendet werden soll will ich, dass er/sie was zum Anhalten hat was ein bissi griffig ist